src/Symfony/Component/String/AbstractString.php

```diff
@@ -426,6 +426,13 @@ public function jsonSerialize(): string
      */
     abstract public function match(string $regexp, int $flags = 0, int $offset = 0): array;
 
+    /**
+     * @param string $to the target newline
+     *
+     * @return static
+     */
+    abstract public function normalizeNewline(string $to = "\n"): self;
```

Comment:
Just for reference purposes: I was trying to find this method in other languages to see if naming matches, but I couldn't find this method in Python or Golang APIs. Maybe it's not that common? Maybe it's hard to implement it well?

Reply:
I did not look for it on other APIs.